tests/dataflow/test_unified_systolic.py

```diff
@@ -6,112 +6,151 @@
 import allo.dataflow as df
 import allo.backend.hls as hls
 import numpy as np
+import pytest
```

Comment:
I would suggest not using pytest for dataflow cases for now

Reply:
Sure, I'll fix it and also aforementioned naming issues.